### PR TITLE
[CC-57] Update Breadcrumb

### PR DESCRIPTION
**Pull request by Audiosutras**
https://caktus.atlassian.net/browse/CC-57

https://caktusgroup.slack.com/archives/C014SUAT8Q2/p1601300469010600?thread_ts=1600995682.002600&cid=C014SUAT8Q2

Populates Breadcrumb with correct text. This PR should be merged after #37 is merged.

### CODE REVIEW

File: cc_licenses/templates/includes/breadcrumbs.html [hunk 1]
```diff
@@ -3,10 +3,11 @@
...
       <li><a href="#">{% trans "Home" %}</a></li>
-      <li><a href="#">{% trans "Child page" %}</a></li>
-      <li><a href="#">{% trans "Child child page" %}</a></li>
-      {% if legalcode %}
-        <li class="is-active"><a href="#" aria-current="page displayed">{% include "includes/snippet/license_medium_title.html" %}</a></li>
+      <li><a href="#">{% trans "License" %}</a></li>
+      {% if 'legalcode' in request.path %}
+        <li class="is-active"><a href="#" aria-current="page displayed">{% trans "Legal Code for" %} {% include "includes/snippet/license_medium_title.html" %}</a></li>
+      {% else %}
+      <li class="is-active"><a href="#" aria-current="page displayed">{% trans "License Deed for" %} {% include "includes/snippet/license_medium_title.html" %}</a></li>
```
Comment (dpoirier) [inline cc_licenses/templates/includes/breadcrumbs.html:10]: It looks like this will display "license deed for" on every page that isn't a license page. (breadcrumbs.html is included unconditionally in base.html.)  Which isn't a big deal right now since we only have those two kinds of pages. We could just put the right breadcrumbs in deed_40.html and legalcode_40_page.html. Or let it go for now. What do you think?

Comment (Audiosutras) [reply cc_licenses/templates/includes/breadcrumbs.html:10]: Thanks, I think we should do what you suggest since if/when we add another page we won't have to hunt down why that page is displaying `License Deed For`.

Comment (Audiosutras) [reply cc_licenses/templates/includes/breadcrumbs.html]: pushed changes

File: cc_licenses/templates/includes/use_of_licenses.html [hunk 2]
```diff
@@ -21,7 +21,7 @@ <h4 class="level is-vcentered b-header">{% trans "Considerations for licensors"
       and conditions of the license they choose before applying it. Licensors should also secure all rights necessary before applying
       our licenses so that the public can reuse the material as expected. Licensors should clearly mark any material not subject to the license.
       This includes other CC-licensed material, or material used under an exception or limitation to copyright.
-      <a href="#">More considerations for licensors.</a>
+      <a href="https://wiki.creativecommons.org/wiki/Considerations_for_licensors_and_licensees#Considerations_for_licensors">More considerations for licensors.</a>
```
Comment (dpoirier) [inline cc_licenses/templates/includes/use_of_licenses.html:24]: It looks as if a couple of unrelated changes in this file snuck into the PR.

File: cc_licenses/templates/base.html [hunk 3]
```diff
@@ -20,7 +20,15 @@
   <main>
     <div class="level padding-{% start %}-big padding-{% end %}-large padding-vertical-normal">
           {% include 'includes/skiplink.html' %}
-          {% include 'includes/breadcrumbs.html' %}
+          <nav class="breadcrumb level-left caption bold" aria-label="breadcrumbs">
+            <ul>
+                <li><a href="#">{% trans "Home" %}</a></li>
+                <li><a href="#">{% trans "License" %}</a></li>
+                {% block breadcrumbs-li %}
+                  <li class="is-active"><a href="#" aria-current="page displayed">{% include "includes/snippet/license_medium_title.html" %}</a></li>
```
Comment (dpoirier) [inline cc_licenses/templates/base.html:28]: It looks like - if I'm reading it all right - the only thing we're changing is the text displayed for the link. Might it make sense to make just that the block, and just have the markup for this item here?  Or maybe not - I'm not sure what this is actually supposed to link *to*, so maybe that'll eventually be changing?

File: cc_licenses/templates/includes/use_of_licenses.html (hunk 2 above)
Comment (Audiosutras) [reply cc_licenses/templates/includes/use_of_licenses.html:24]: yes, do you want me to remove this link? I put it here from my chat with vocab team.

File: cc_licenses/templates/base.html (hunk 3 above)
Comment (Audiosutras) [reply cc_licenses/templates/base.html:28]: done

File: cc_licenses/templates/includes/use_of_licenses.html (hunk 2 above)
Comment (dpoirier) [reply cc_licenses/templates/includes/use_of_licenses.html:24]: If it was intentional, not a big deal.

**Review by dpoirier — APPROVED** [review]

File: cc_licenses/templates/includes/use_of_licenses.html (hunk 2 above)
Comment (Audiosutras) [reply cc_licenses/templates/includes/use_of_licenses.html:24]: got it.
